### DISCUSSION TITLE
Slider Power independent?

### DISCUSSION THREAD

**Discussion by mmmmm009** [arendst/Tasmota#20828] [Q&A]
Hi,
I have the following setup:
![tasmo](https://github.com/arendst/Tasmota/assets/72743330/993b2db1-0556-4a3d-8e6e-06134cab0fed)

When POWER2 is OFF (and Channel2 slider in left possition-Channel2 = 0) and when I move Channel2 slider to the right then POWER2 changes to ON. 
Can I detach Slider/Channel2 from POWER2?
So...
By changing Channel2-slider from 0 to whatever POWER2 changes from OFF to ON.
THX! :)

**Comment by sfromis** [comment]
In general, the deliberately rudimentary Tasmota Web UI is merely meant for admin/test purposes, not as a "device dashboard" for regular operation, hence there are very few options for tweaking how it works.

And when you seem to be using `SetOption68` to split PWM into channels instead of a light, that likely means that `SetOption20`, to allow update of light-related values without this turning on power, will not work either.

**Comment by mmmmm009** [comment]
> In general, the deliberately rudimentary Tasmota Web UI is merely meant for admin/test purposes, not as a "device dashboard" for regular operation, hence there are very few options for tweaking how it works.

That's a pitty. Tasmota is a nice and powerfull thing. Would be nice if it was a bit more...adjustable.

> And when you seem to be using SetOption68 to split PWM into channels instead of a light, that likely means that SetOption20, to allow update of light-related values without this turning on power, will not work either.

`SetOption68 1` and `SetOption20 1` totally disabled the slider (it doesn't update ChannelX value anymore). unless there is any other parameter that it still changes?
